### DISCUSSION TITLE
Programmatically modify youngs modulus in flexcomp meshes

### DISCUSSION THREAD

**Discussion by Nick20500** [google-deepmind/mujoco#2134] [Asking for Help]
### Intro

Hi!

I am a undergrad student at ETH Zurich, I use MuJoCo for my research on Multimaterial deformable objects.

### My setup

I use MuJoCo 3.2.3 with python bindings.

### My question

Am I able to programmatically modify the youngs modulus of the bottle object in this example: 
[example.zip](https://github.com/user-attachments/files/17342640/example.zip)

I assume, it should be somehow possible with the new "mjSpec feature" mentioned here: https://mujoco.readthedocs.io/en/stable/python.html#pymodeledit

But I can't see if this is already implemented and the documentation how to use is missing or if it isn't implemented yet

### Confirmations

- [X] I searched the [latest documentation](https://mujoco.readthedocs.io/en/latest/overview.html) thoroughly before posting.
- [X] I searched previous [Issues](https://github.com/google-deepmind/mujoco/issues) and [Discussions](https://github.com/google-deepmind/mujoco/discussions), I am certain this has not been raised before.

**Comment by quagla** [comment]
Hi, this will be available in a few days starting from the next release!
